<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadElements</class>
 <widget class="QDialog" name="LoadElements">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load WaterIng Data</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="scenarioLabel">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Scenario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="newScenarioCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New Scenario</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="projectLabel">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="newProjectCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New Project</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="scenarios_box"/>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="projectLabel3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Project Directory</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
   <item row="10" column="1">
    <widget class="QgsFileWidget" name="newShpDirectory">
     <property name="storageMode">
      <enum>QgsFileWidget::GetDirectory</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="projects_box"/>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
   <item row="8" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="19" column="1">
    <widget class="QTabWidget" name="container">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Load_4">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <attribute name="title">
       <string>Load</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="new_project_name">
         <property name="placeholderText">
          <string>Project Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="create_scenario">
         <property name="placeholderText">
          <string>Scenario Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorLogin_4">
         <property name="styleSheet">
          <string notr="true">color:red</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newProjectBtn">
         <property name="text">
          <string>Load WaterIng Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanced_4">
      <attribute name="title">
       <string>Clone</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QCheckBox" name="cloneCheckBox">
           <property name="text">
            <string>Clone Scenario</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="clone_title">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Clone into project:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="clone_box"/>
       </item>
       <item>
        <widget class="QLineEdit" name="cloned_scenario_name">
         <property name="placeholderText">
          <string>Cloned Scenario Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cloneScenarioBtn">
         <property name="text">
          <string>Clone</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Merge</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="mergeCheckBox">
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Merge Scenario</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="projectLabel_5">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>Project</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="merge_projects_box"/>
       </item>
       <item>
        <widget class="QLabel" name="scenarioLabel_4">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>Scenario</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="merge_scenarios_box"/>
       </item>
       <item>
        <widget class="QLabel" name="merge_title">
         <property name="text">
          <string>Merge into project:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="merge_box"/>
       </item>
       <item>
        <widget class="QLineEdit" name="merged_scenario_name">
         <property name="placeholderText">
          <string>Merged Scenario Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mergeScenariosBtn">
         <property name="text">
          <string>Merge</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
